doc/source/poppy_config.rst

```diff
@@ -0,0 +1,72 @@
+
+=========================
+Poppy Configuration Notes
+=========================
+
+Description
+-----------
+
+Please find below various poppy conf file configuration beside which in comments you will find the necessity of those configurations.
+These are some of the important configurations and also some regular configurations, not all the configuration information are explained below.
+
+Default configuration
+---------------------
+
+Make sure to keep the logging.conf in the correct directory
+
+[DEFAULT]
+verbose = True 			#To get verbosity for poppy server and worker
+debug = True 			#to get debug logs for poppy server and worker
+base_url = http://0.0.0.0:8888	#To have accessibility from anywhere
+
+log_config_append = /Users/<user>/.poppy/logging.conf   
+
+
+drivers configuration
+---------------------
+
+[drivers]
+providers = akamai 		#We currently have only Akamai as the provider
+
+
+storage configuration
+---------------------
+We currently use cassandra as our database.
+Set the migrations path correctly, to the location where you will find the migrations under the source code
```

Comment:
~This really doesn't clear anything up if you have no idea what this is and where it's supposed to be.~

Maybe add something like:

The migrations folder is located `poppy/storage/cassandra/migrations` (relative to the root `poppy` folder).

For example, if I cloned poppy to my home directory, the full path would be `/home/heiths/poppy/poppy/storage/cassandra/migrations`.

With `automatic_schema_migration` set to `true`, you should see the migrations run automatically after the first request following a fresh cassandra container.